src/utils/scores.ts

```diff
@@ -81,3 +81,54 @@ export async function getIssueEventsForTeam(team) {
 
   return issues;
 }
+
+export async function getDiscussionEvents() {
+  const discussionsQuery = `
+    SELECT
+      discussions.target_name as title,
+      discussions.repository as repository,
+      discussions.object_id as discussion_number,
+      COUNT(discussions.target_name) as num_comments,
+    FROM
+      \`open_source.github_events\` AS discussions
+    WHERE
+      discussions.type = 'discussion_comment'
+      AND timestamp_diff(
+        CURRENT_TIMESTAMP(),
+        discussions.created_at,
+        day
+      ) <= 7
+    GROUP BY discussions.target_name, discussions.repository, discussions.object_id
+    ORDER BY num_comments DESC
+    ;`;
+
+  const [discussions] = await bigqueryClient.query(discussionsQuery);
+
+  const discussionCommentersQuery = `
+    SELECT
+      discussions.username as username,
+      COUNT(discussions.username) as num_comments,
+    FROM
+      \`open_source.github_events\` AS discussions
+    WHERE
+      discussions.type = 'discussion_comment'
+      AND timestamp_diff(
+        CURRENT_TIMESTAMP(),
+        discussions.created_at,
+        day
+      ) <= 7
+      AND discussions.user_type != 'external'
+      AND discussions.user_type != 'bot'
+    GROUP BY discussions.username
+    ORDER BY num_comments DESC
```

Comment:
Out of scope for this PR, but I wonder if there is some standard way of running a local in-memory instance of SQLite to query against, or something similar? Otherwise we end up with a ton of untested code: we validate that the exact text of our SQL code is what we expect, but not what that SQL code actually does (ie, does it get the exact rows in the table we want it to?).

I could imagine using in-memory SQLite as our backend, since this looks like bog-standard SQL with no BigQuery specific extensions in it.

Just a thought - not a blocker by any means, and definitely not a thing to do in this PR, but something to think about, since we've had a few cases of queries not returning the data set we expect them to.

Reply:
Yeah, that'd be nice to just to get some validation that it's valid sql. Adding to the backlog of things to do for eng-pipes:
https://github.com/getsentry/eng-pipes/issues/676

Reply:
but yes, the dev environment setup for BigQuery leaves much to be desired....

I don't think most people who have worked in eng-pipes have even hooked up their env to BigQuery